docs/monitoring.md

```diff
@@ -114,16 +114,16 @@ The history server can be configured as follows:
     <td>spark.history.retainedApplications</td>
     <td>50</td>
     <td>
-      The number of application UIs to retain. If this cap is exceeded, then the oldest
-      applications will be removed.
+      The number of application UIs to retain in the cache. If this cap is exceeded, then the oldest
+      applications will be removed from the cache.
```

Comment:
Kind of a nit here but perhaps we should change this to "The number of applications to retain in the cache".  I'm wondering if UIs could still be confusing with the number displayed.  And perhaps after the last sentence add something like: If an application is not in the cache, it will have to be loaded from disk if its accessed from the UI.